code/index.html

```diff
@@ -1,30 +1,107 @@
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
+    <title>Rent-a-classic-VW</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Raleway&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <!-- video or image as a header is cool :) went for image. Cool enough :)-->
+    <div class="hero-image">
+        <div class="hero-text">
+            <h1>RENT A CLASSIC VW</h1>
+            <p>Dreaming of those perfect insta-moments? Curious about the vanlife- or vw-community?</p>
+            <p>Rent one of our cars and make YOUR dreams come true.
+            </p>
+        </div>
+    </div>
+
+
+    <main>
+        <!-- Signup form -->
+        <section class="booking">
+            <div>
+                <h2>Booking</h2>
+            </div>
+            <form class="form" action="http://httpbin.org/anything" method="POST">
+                <div class="form-group">
+                    <label for="name">Name:</label>
+                    <input id="name" type="text" name="name" placeholder="Your full name here" name="nameValue" required>
+                </div>
+                <div class="form-group">
+                    <label for="email">E-mail:</label>
+                    <input id="email" type="email" name="email" placeholder="Your e-mail adress here" required>
+                </div>
+                <div class="form-group">
+                    <label for="car">VW Baywindow Bus</label>
+                    <input id="car" type="radio" name="car" value="vwBaywindowBus"></div>
+                <div class="form-group">
+                    <label for="car">VW Beetle</label>
+                    <input id="car" type="radio" name="car" value="vwBeetle">
+                </div>
+                <div class="form-group">
+                    <label for="numberofadults">How many adults?</label>
+                    <input id="numberOfAdults" type="number" name="numberofadults" min="0" max="3"></div>
+                <div class="form-group">
+                    <label for="numberofkids">How many kids?</label>
+                    <input id="numberOfKids" type="number" name="numberofkids" min="0" max="2">
+                </div>
+                <div class="form-group"><button id="submit" type="submit" name="submit">Submit</button></div>
+            </form>
+        </section>
+
+        <section class="accordion">
+            <!-- Accordion -->
+            <h1>FAQ</h1>
+
+            <div class="question" id="section1">
+
+                <h3>What do we do if we have a breakdown?
+                </h3>
+            </div>
+            <div class="answer">
+                <p>They are old cars, you WILL have breakdowns. Call us first, it's rarely something advanced. If we can't fix it then we call Assistanskåren.</p>
+            </div>
+
+            <div id="section2" class="question">
```

Comment:
It's a super small issue, but one question above your order of attributes is `class` and then `id`, but over here it's opposite